src/main/java/com/dsrm/dsrmbackend/controllers/UserController.java

```diff
@@ -46,8 +46,15 @@ ResponseEntity<UserDTO> getUser(@PathVariable Long id){
 
     @GetMapping(value = "/users")
     @PageableAsQueryParam
-    public ResponseEntity<Page<UserDTO>> readUsers(@Parameter(hidden = true) Pageable pageable) {
+    ResponseEntity<Page<UserDTO>> readUsers(@Parameter(hidden = true) Pageable pageable) {
         Page<User> userPage = userService.getUsers(pageable);
         return new ResponseEntity<>(userPage.map(userMapper::toUserDTO),HttpStatus.OK);
     }
+
+    @PatchMapping(value = "/users/{id}/roles")
+    ResponseEntity<UserDTO> partialUpdateUser(@PathVariable Long id, @RequestBody UserRolesOnlyDTO userRolesOnlyDTO){
```

Comment:
I believe it should be void because in path we don't want to return whole resource.